htdocs/wp-content/themes/owc-formulieren/templates/partials/sidebar-mijn-zaken.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Template partial for the NLDS Den Haag Sidebar navigation.
+ * @design https://nl-design-system.github.io/denhaag/?path=/docs/css-navigation-sidenav--default-story
+ * @author Paul van Impelen <paul@acato.nl>
+ */
+
+?>
+<nav class="denhaag-sidenav">
+    <ul class="denhaag-sidenav__list">
+        <li class="denhaag-sidenav__item">
+            <a href="/" aria-current="page" class="denhaag-sidenav__link denhaag-sidenav__link--current">
+                <svg
+                    width="1em" height="1em" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"
+                    class="denhaag-icon" focusable="false" aria-hidden="true" shape-rendering="auto">
+                    <path
+                        d="M3 5a2 2 0 012-2h4a2 2 0 012 2v4a2 2 0 01-2 2H5a2 2 0 01-2-2V5zm6 0H5v4h4V5zm4 0a2 2 0 012-2h4a2 2 0 012 2v4a2 2 0 01-2 2h-4a2 2 0 01-2-2V5zm6 0h-4v4h4V5zM3 15a2 2 0 012-2h4a2 2 0 012 2v4a2 2 0 01-2 2H5a2 2 0 01-2-2v-4zm6 0H5v4h4v-4zm4 0a2 2 0 012-2h4a2 2 0 012 2v4a2 2 0 01-2 2h-4a2 2 0 01-2-2v-4zm6 0h-4v4h4v-4z"
+                        fill="currentColor" />
+                </svg>
+                Overzicht
+            </a>
+        </li>
+        <li class="denhaag-sidenav__item">
+            <a href="#mijn-berichten" class="denhaag-sidenav__link">
```

Comment:
Kun je alle links in deze file voor nu hardcoded naar de desbetreffende slug verwijzen? 

`<?php echo home_url('mijn-berichten'); ?>`